source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -61,11 +61,15 @@ Test the configuration
 
 #. Remove the Vim package to fix the vulnerability. Wait for syscollector to run a new scan.
 
-Visualize the alerts
---------------------
+If the syscollector scan has finished but you don't find the changes reflected in the Wazuh dashboard, it might be the case that the indexation cool down period for that agent is preventing the indexation. This 30 minutes time window can't be changed.
```

Comment:
Maybe we could mention that this feature is named "abuseControl" so the user can make that connection with the logs

Reply:
I've updated the sentence.